phpunit.xml.dist

```diff
@@ -12,4 +12,8 @@
         </testsuite>
     </testsuites>
 
+    <listeners>
+       <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
```

Comment:
is this related?

Reply:
Yes, you cannot use assertions on the deprecations without this.

Reply:
ah, ok. cool !